doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -30,8 +30,8 @@ enable more features, such as distributed memory execution.
 
          tar xzf chapel-1.23.0.tar.gz
 
-   b. Make sure that your shell is in the directory containing
-      QUICKSTART.rst, for example:
+   b. Make sure that your shell is in the directory you just created by
```

Comment:
While we're here, I wonder whether it would be even clearer to drop the reference to "your shell".  E.g., "Make sure that you are in the directory you just created by unpacking the source release, for example:"

Reply:
I don't feel strongly one way or the other, so I can do that unless someone objects vehemently

Reply:
I tend to agree with @bradcray 
Dropping "your shell" makes it sound a lot more organic